docs/docs/how-tos/nebari-gcp.md

```diff
@@ -0,0 +1,155 @@
+
+Follow [these detailed instructions](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account) to create a Google Service Account with **Owner** level permissions over the project created in the previous step. For more information about roles and permissions, see the [Google Cloud Platform IAM documentation](https://cloud.google.com/iam/docs/choose-predefined-roles).
+
+After you create your service account, download the [service account key](https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts.keys) file following the instructions below:
+
+- From the [service account key page in the Cloud Console](https://console.cloud.google.com/iam-admin/serviceaccounts) choose your recently created project;
+- Select your service account from the list;
+- Select the "Keys" tab;
+- In the drop down menu, select "Create new key";
+- Leave the "Key Type" as `JSON`;
+- Click "Create" to create the key and save the key file to your system.
+
+Store this credentials file in a well known location and make sure to set yourself exclusive permissions. You can change the file permissions by running the command `chmod 600 <filename>` on your terminal.
+
+:::warning Warning
+The service account key file provides access to your GCP project. It should be treated like any other secret credentials. Specifically, it should never be checked into source control.
+:::
+
+By default, Nebari will try to use the credentials associated with the current Google Cloud infrastructure/environment for authentication. Provide authentication credentials to Nebari by setting the following environment variables:
+
+```bash
+export GOOGLE_CREDENTIALS="path/to/JSON/file/with/credentials"
+export PROJECT_ID="projectIDName"
+```
+The **project ID** information can be found at the Google Console homepage, under **Project info**.
+
+This variables applies only to your current shell session. If you want the variable to apply to future shell sessions, set the variable in your shell startup file, for example in the `~/.bashrc` or `~/.profile` file.
+
+
+:::note
+The following steps assumes you have completed the [Install Nebari](/started/installing-nebari) section, has confirmed that `nebari` is successfully installed in your environment and opted for **GCP** as your cloud provider as well as installed and initialized `gcloud`. If you had any issues during the installation, please visit our getting started [troubleshooting](/started/troubleshooting) section for extra advice.
+:::
+
+## Nebari Initialize
+
+Great, you’ve gone through the [Nebari Installation](/started/installing-nebari.md) and [Authentication setup](#authentication) steps, and have ensured that all the necessary environment variables have been properly set. It is time to initialize and deploy Nebari!
+
+In your terminal, we advise you to start by creating a new project folder. Here, we will name the new folder as `data`:
```

Comment:
Can we find use another folder name? Perhaps we can just reuse the `projectname`?

Reply:
no problem, what about `nebari` then?